_data/rules.yml

```diff
@@ -538,6 +538,34 @@ groups:
           - name: percona/mongodb_exporter
             doc_url: https://github.com/percona/mongodb_exporter
             rules:
+              - name: MongoDB Down
+                description: MongoDB instance is down
+                query: 'mongodb_up == 0'
+                severity: critical
+              - name: MongoDB replication lag
+                description: Mongodb replication lag is more than 10s
+                query: 'avg(mongodb_mongod_replset_member_optime_date{state="PRIMARY"}) - avg(mongodb_mongod_replset_member_optime_date{state="SECONDARY"})  > 10'
+                severity: critical
+              - name: MongoDB replication headroom
+                description: MongoDB replication headroom is <= 0
+                query: '(avg(mongodb_mongod_replset_oplog_tail_timestamp - mongodb_mongod_replset_oplog_head_timestamp) - (avg(mongodb_mongod_replset_member_optime_date{state="PRIMARY"}) - avg(mongodb_mongod_replset_member_optime_date{state="SECONDARY"}))) <= 0'
```

Comment:
`mongodb_mongod_replset_oplog_tail_timestamp` and `mongodb_mongod_replset_oplog_head_timestamp` doesn't have any label about their replica set, so I can only aggregate them by their instance

Reply:
Hi @samber @yasharne I was about to open a PR but find out this one which is more or less related. 

My PR was about removing the alerts based on mongodb_mongod_replset_oplog* since they have been [deprecated](https://github.com/percona/mongodb_exporter#important-note).

Will still proceed with my PR but it think it explains the lack of replica set label